.github/actions/issue-validator/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "issue-validator",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
You can remove name/version/descritpion/keywords/author/license and just add `"private": true`